test_runner/src/main/kotlin/ftl/reports/xml/JUnitXml.kt

```diff
@@ -15,7 +15,7 @@ private val xmlMapper = XmlMapper(xmlModule)
     .registerModules(KotlinModule())
     .configure(FAIL_ON_UNKNOWN_PROPERTIES, false)
 
-private val xmlPrettyWriter = xmlMapper.writerWithDefaultPrettyPrinter()
+internal val xmlPrettyWriter = xmlMapper.writerWithDefaultPrettyPrinter()
 
 private fun xmlBytes(path: Path): ByteArray {
     if (!path.toFile().exists()) RuntimeException("$path doesn't exist!")
```

Comment:
I am totally aware it is not part of your PR (so feel free to skip this comment). I think there is `throw` missing

Reply:
Good checkup. I can fix it in this PR, no problem.